<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>50000</width>
    <height>50000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TankBattles. Редактор карт</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="field">
      <property name="minimumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>70</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>70</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="tools_layout">
        <item>
         <widget class="QFrame" name="tools_frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>183</width>
            <height>148</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QGroupBox" name="tools_groupbox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>167</width>
             <height>148</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(0, 0, 0);</string>
           </property>
           <property name="title">
            <string>Инструменты</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <widget class="QRadioButton" name="ubr_btn">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>25</y>
              <width>141</width>
              <height>28</height>
             </rect>
            </property>
            <property name="text">
             <string>Неразрушаеммый
блок</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="emp_btn">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>59</y>
              <width>106</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>Пустой блок</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="spwn_btn">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>118</y>
              <width>105</width>
              <height>19</height>
             </rect>
            </property>
            <property name="text">
             <string>Блок спавна</string>
            </property>
           </widget>
           <widget class="QRadioButton" name="br_btn">
            <property name="geometry">
             <rect>
              <x>17</x>
              <y>84</y>
              <width>124</width>
              <height>28</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="text">
             <string>Разрушаеммый
 блок</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="size_layout">
        <item>
         <widget class="QFrame" name="size_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>183</width>
            <height>126</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QGroupBox" name="size_groupbox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>170</width>
             <height>116</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>116</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>170</width>
             <height>116</height>
            </size>
           </property>
           <property name="title">
            <string>Размер</string>
           </property>
           <widget class="QSpinBox" name="xspin">
            <property name="geometry">
             <rect>
              <x>39</x>
              <y>30</y>
              <width>106</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
           <widget class="QLabel" name="X_label">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>33</y>
              <width>21</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="yspin">
            <property name="geometry">
             <rect>
              <x>39</x>
              <y>57</y>
              <width>106</width>
              <height>24</height>
             </rect>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
           <widget class="QLabel" name="Y_label">
            <property name="geometry">
             <rect>
              <x>16</x>
              <y>60</y>
              <width>21</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
           <widget class="QPushButton" name="apply_button">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>90</y>
              <width>141</width>
              <height>27</height>
             </rect>
            </property>
            <property name="text">
             <string>Применить</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="scale_layout">
          <item>
           <widget class="QFrame" name="scale_frame">
            <property name="maximumSize">
             <size>
              <width>187</width>
              <height>56</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QGroupBox" name="scale_groupbox">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>0</y>
               <width>161</width>
               <height>61</height>
              </rect>
             </property>
             <property name="title">
              <string>Масштаб</string>
             </property>
             <widget class="QSlider" name="scale_slide">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>30</y>
                <width>160</width>
                <height>18</height>
               </rect>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>201</number>
              </property>
              <property name="value">
               <number>70</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="preview">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Предварительный
 просмотр</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>Фаил</string>
    </property>
    <addaction name="New"/>
    <addaction name="Load"/>
    <addaction name="Save"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="Help"/>
    <addaction name="separator"/>
    <addaction name="About"/>
   </widget>
   <addaction name="file"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Load">
   <property name="text">
    <string>Загрузить</string>
   </property>
   <property name="toolTip">
    <string>Загрузить карту</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить карту</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="New">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="toolTip">
    <string>Создать новый фаил</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="Help">
   <property name="text">
    <string>Краткая информация</string>
   </property>
   <property name="toolTip">
    <string>Краткая справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
